src/clj/web/chat.clj

```diff
@@ -30,11 +30,29 @@
         msg-cnt (mc/count db msg-collection {:username username :date {"$gt" start-date}})]
     (< msg-cnt max-cnt)))
 
-(defn- insert-msg [{{{:keys [username emailhash]} :user} :ring-req
-                    {:keys [:channel :msg]} :?data :as event}]
+(let [alpha (into #{} "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
```

Comment:
Why is this both upper and lower case? Is it not better to match just on lowercase?

Reply:
Yeah, that's probably a good point. I just copy-pasted this from a gist I found, so I can definitely make that change.

Reply:
This actually doesn't work, because unless we want to call "lower-case" on all messages before checking them, we'll miss any capital letters. Easier to convert to rot13 and then ignore-case in the regex.